app/models/opinion.rb

```diff
@@ -0,0 +1,8 @@
+class Opinion < ApplicationRecord
+
+  belongs_to :user
+  belongs_to :comment
+
+  validates :user, uniqueness: { scope: :opinion }
```

Comment:
To dotyczy komentarza a nie opini o komentarzu

Reply:
Racja, w drugim commicie już zmienione.